frontend/src/App.vue

```diff
@@ -196,4 +196,9 @@ export default {
   .viewer-open {
     padding-right: 0 !important;
   }
```

Comment:
Can this go in the datepicker component?

Reply:
@mightycox Sure https://github.com/bcgov/EDUC-STUDENT-ADMIN/pull/1686/commits/223a5fbdbdc0cb3ce1e8d8ae1e02767480a0b83c